File: API/Models/UsuarioLojaModel.cs
namespace API_Visitatus.Models
{
    public class UsuarioLojaModel
    {
        public long UsuCodi { get; set; }
        public string? UsuNome { get; set; }
        public string? UsuNCIM { get; set; }
        public string? UsuNCel { get; set; }
        public string? UsuEmai { get; set; }
        public DateTime UsuNasc { get; set; }
        public bool UsuStat { get; set; }
        public int PerCodi { get; set; }
        public string? PerNome { get; set; }
        public bool PeUStat { get; set; }
    }
}

File: Sistema/src/app/models/UsuarioLoja.Model.ts
export class UsuarioLojaModel {
  UsuCodi: number;
  UsuNome: string;
  UsuNCIM: string;
  UsuNCel: string;
  UsuEmai: string;
  UsuNasc: Date | null;
  UsuStat: boolean;
  PerCodi: number;
  PerNome: string;
  PeUStat: boolean;

  constructor() {
    this.UsuCodi = 0;
    this.UsuNome = '';
    this.UsuNCIM = '';
    this.UsuNCel = '';
    this.UsuEmai = '';
    this.UsuNasc = null;
    this.UsuStat = false;
    this.PerCodi = 0;
    this.PerNome = '';
    this.PeUStat = false;
  }
}
